tox.ini

```diff
@@ -136,7 +136,7 @@ commands_pre=
     pip install -r {envtmpdir}/requirements.txt
     pip freeze
 commands=
-    pytest astropy/astropy/io/misc/asdf --open-files --run-slow --remote-data
+    pytest astropy/astropy/io/misc/asdf --open-files --run-slow --remote-data -W "ignore:The property AsdfFile.blocks has been deprecated:asdf.exceptions.AsdfDeprecationWarning:asdf.asdf:661"
```

Comment:
@WilliamJamieson What are your thoughts on this as a way to keep our downstream CI from failing until astropy is updated to ignore this new deprecation?

Reply:
This is fine